packages/vite/index.d.ts

```diff
@@ -0,0 +1,5 @@
+export * from './src/resolver.js';
```

Comment:
Was this supposed to be comitted? 🙃

Reply:
Yes. This is working around the fact that our packages all compile to commonjs and the vite config is a real node ES module. Both the implementation and the types go through these top-level reexports, which happens to make the cjs interop work out ok.

Reply:
Someone please do the big yak shave to get all our packages dual compiling as CJS and ESM so I don't have to. 😅